client/src/elm/Pages/Prenatal/Utils.elm

```diff
@@ -2393,6 +2468,11 @@ resolveVitaminADosageAndIcon currentDate person =
     Just ( "200,000 IU", "icon-pills" )
 
 
+resolveParacetamolDosageAndIcon : NominalDate -> Person -> Maybe ( String, String )
+resolveParacetamolDosageAndIcon currentDate person =
```

Comment:
Those variables don't seem to be used here, do you want to keep them?

Reply:
@nedSaf 

It's in use:

![Selection_1030](https://user-images.githubusercontent.com/17882245/184103351-4136c8bd-b2f5-43c1-898d-3598537ccbc7.png)

Reply:
Not the function, the variables of this function: `currentDate` & `person`

Reply:
@nedSaf Ah, I see.
We have many places where passed variables are not in use.
No need to change - we pay nothing for passing them, and may need them in future.

Reply:
👍 Okay, sounds good.